### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by TsubasaK111**
@KajiwaraMidori review me!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ This is the Enjoy Vegan Japan website repo.
 
 ## Get Started
 - Install all requirements
+- Open Apache's `httpd.conf` and
```
Comment (KajiwaraMidori) [inline README.md]: delete

File: README.md [hunk 2]
```diff
@@ -7,6 +7,8 @@ This is the Enjoy Vegan Japan website repo.
...
+  set `DocumentRoot` and `Directory` to this repo's `index.php`
```
Comment (KajiwaraMidori) [inline README.md]: delete

File: README.md [hunk 3]
```diff
@@ -28,7 +31,13 @@ This is the Enjoy Vegan Japan website repo.
 - MacOS
   - [How to install Wordpress on MacOS](https://qiita.com/purio/items/416a6762583c087ece61)    
   - [How to update MacOS PHP to the latest version](https://php-osx.liip.ch/)
+<<<<<<< HEAD
```
Comment (KajiwaraMidori) [inline README.md]: delete

File: README.md [hunk 4]
```diff
@@ -28,7 +31,13 @@ This is the Enjoy Vegan Japan website repo.
...
+  - Updates and theme/plugin installs will fail to work at first.  
+    Set Apache's `httpd.conf`'s `User` setting to the Mac User,  
+    and `sudo apachectl restart`
+=======
```
Comment (KajiwaraMidori) [inline README.md]: delete

File: README.md [hunk 5]
```diff
@@ -28,7 +31,13 @@ This is the Enjoy Vegan Japan website repo.
...
   - Open Apache's `/etc/apache2/httpd.conf` and follow 2 steps
     comment out 'LoadModule php5_module libexec/apache2/libphp5.so'
     set `DocumentRoot` and `Directory` to this repo's `index.php`
 
+>>>>>>> master
```
Comment (KajiwaraMidori) [inline README.md]: delete

**Review by KajiwaraMidori — CHANGES_REQUESTED** [review]

**Comment by TsubasaK111** [comment]
どや

**Review by KajiwaraMidori — APPROVED** [review]

**Pull request merged**